gui/default/syncthing/core/syncthingController.js

```diff
@@ -2662,6 +2662,15 @@ angular.module('syncthing.core')
                          + '&device=' + encodeURIComponent(deviceID));
         };
 
+        $scope.sharesFolderEncrypted = function (folderID, deviceID) {
```

Comment:
I think the naming scheme of the following functions is easier to understand (e.g. `folderHasUnacceptedDevices`), so something like `folderIsSharedEncrypted`.

Reply:
Done.